docs/application/dotnet/guides/nui/resources.md

```diff
@@ -0,0 +1,87 @@
+# Resources
+
+NUI provides several ways to handle resource images.
+This tutorial describes the use of Image resources in NUI.
+
+## Overview
+
+Resources in NUI can apply Images or 3D Models.
+
+The common method to access these resources is through Visuals.
+Controls use these visuals to display what is required.
+
+See [ImageView tutorial](imageview.md).
+
+## Loading images
+
+Images are defined by an url which can be local or remote (internet).
+The following formats are supported :
+
+| Supported image formats | extentsion |
+|------------------------ | ---------- |
+| Bitmap                  | `.bmp`     |
+| Gif                     | `.gif`     |
+| Jpeg                    | `.jpeg`    |
+| Khronos Texture         | `.ktx`     |
+| PNG                     | `.png`     |
+| Mircosoft Icons         | `.ico`     |
+| WAP Bitmap              | `.wbmp`    |
+
+Controls should process the provided URL and internally create the matching Image Visual.
+Visual creation automatically determines whether the image is N_PATCH, SVG, or GIF, which is not a regular image from the image data.
+
+### Image Properties
+
+| Property Name        | Description       |
+|----------------------|-------------------|
+| `AlphaMaskURL`       | url to an image that will mask the main content image. |
+| `CropToMask`         | Flag to determine if main content image should crop to match mask size. |
+| `FittingMode`        | By default the image will shrink to fit the DesiredHeight and DesiredWidth, other modes available. |
+| `SamplingMode`       | The type of sampling to be used. Default is Box. |
+| `DesiredWidth`       | The width you would like the Image to be.  Affected by FittingMode.       |
+| `DesiredHeight`      | The height you would like the Image to be. Affected by FittingMode.       |
+| `SynchronousLoading` | If loading should block the execution thread, disabled by Default.        |
+| `PixelArea`          | Can be set to display only a portion of the Image.           |
+| `WrapModeU`          | Define how sampling should behave for u coordinate. |
+| `WrapModeV`          | Define how sampling should behave for v coordinate. |
+
+
+### N-Patch only properties
```

Comment:
N-Patch only properties
Maintain uniformity in the heading

Reply:
I hope to get the suggestion for that heading.